1_OCaml/TP/TP1.ipynb

```diff
@@ -17,17 +17,68 @@
     "\n",
```

Comment:
<p>Il faut ;; à la fin de la définition de ta fonction globale (ou in pour l'utiliser tout de suite, comme fonction définie localement)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mp2i-fsm/mp2i-2021/pull/46/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>